### PR TITLE
Gidk 1z9fp8 new changes test

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 1]
```diff
@@ -264,8 +264,8 @@ <h3 class="ledger-main-head">
                             <div class="px-3 w-100">
                                 <table class="ledger-head transaction-mapped mt-0" width="100%" colspan="100%">
                                     <tr>
-                                        <th colspan="6" class="pl-15 pr-2 font-14">Debit <span class="pull-right text-gray">Dr</span></th>
-                                        <th colspan="6" class="pr-15 pl-1 font-14">Credit <span class="pull-right text-gray">Cr</span></th>
+                                        <th colspan="6" class="pd-l15 pr-2 font-14">Debit <span class="pull-right text-gray">Dr</span></th>
+                                        <th colspan="6" class="pd-r15 pl-1 font-14">Credit <span class="pull-right text-gray">Cr</span></th>
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/ledger/ledger.component.html:268]: check for this pl-1 also, if it is working fine

File: apps/web-giddh/src/app/new-inventory/component/adjust-group/adjust-group.component.html [hunk 2]
```diff
@@ -1,14 +1,14 @@
 <div class="relative adjust-group-page">
     <hamburger-menu [pageHeading]="'Adjust group'"></hamburger-menu>
-    <i class="icon-cross pull-right pd cursor-pointer" [routerLink]="['/pages/new-inventory/adjust-inventory']"></i>
-
+    <!-- <i class="icon-cross pull-right pd cursor-pointer" [routerLink]="['/pages/new-inventory/adjust-inventory']"></i> -->
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/new-inventory/component/adjust-group/adjust-group.component.html:3]: remove comment if not required

File: apps/web-giddh/src/app/new-inventory/component/create-combo/create-combo.component.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
     <div class="header-create-unit">
         <button id="close" (click)="closeAsidePane($event)"><i class="icon-cross"></i></button>
         <div class="form-header">
-            <div class="back-button mr-15 p-05"><i class="icon-back-arrow"></i></div>
+            <div class="back-button mr-r15 p-05"><i class="icon-back-arrow"></i></div>
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/new-inventory/component/create-combo/create-combo.component.html:5]: p-05 is not a class, please check

File: apps/web-giddh/src/app/new-inventory/component/create-new-item/create-new-item.component.html [hunk 4]
```diff
@@ -2,7 +2,7 @@
     <div class="header-create-group">
         <button id="close" (click)="closeAsidePane($event)"><i class="icon-cross"></i></button>
         <div class="form-header">
-            <div class="back-button mr-15 p-05"><i class="icon-back-arrow"></i></div>
+            <div class="back-button mr-r15 p-05"><i class="icon-back-arrow"></i></div>
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/new-inventory/component/create-new-item/create-new-item.component.html:5]: p-05 is not a class, please check

File: apps/web-giddh/src/app/new-inventory/component/create-unit/create-unit.component.html [hunk 5]
```diff
@@ -2,7 +2,7 @@
     <div class="header-create-unit">
         <button id="close" (click)="closeAsidePane($event)"><i class="icon-cross"></i></button>
         <div class="form-header">
-            <div class="back-button mr-15 p-05"><i class="icon-back-arrow"></i></div>
+            <div class="back-button mr-r15 p-05"><i class="icon-back-arrow"></i></div>
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/new-inventory/component/create-unit/create-unit.component.html:5]: p-05 is not a class

File: apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.scss [hunk 6]
```diff
@@ -19,6 +19,18 @@
                 }
             }
         }
+        .more-add-button{
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.scss:22]: This css is already available in about-combo-details.component.scss, please don't create duplicate css, move this to common file.

File: apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts [hunk 7]
```diff
@@ -99,6 +101,10 @@ export class AboutGroupDetailComponent implements OnInit {
             this.isMobileScreen = result.matches;
         });
     }
+    /* It will show/hide hsn code field */
+    selectCode(isHSN) {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts:105]:
```suggestion
    public selectCode(isHSN: any): void {
```

File: apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.scss [hunk 8]
```diff
@@ -23,6 +23,16 @@
             background-color: $white;
             width: 36px;
         }
+        .icon-undo{
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.scss:26]: duplicate css, remove this

File: apps/web-giddh/src/app/new-inventory/component/create-new-group/create-new-group.component.ts [hunk 9]
```diff
@@ -27,4 +28,8 @@ export class CreateNewGroupComponent implements OnInit {
         /* added image path */
         this.imgPath = (isElectron || isCordova) ? 'assets/images/' : AppUrl + APP_FOLDER + 'assets/images/';
     }
+    /* It will show/hide hsn code field */
+     selectCode(isHSN) {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/create-new-group/create-new-group.component.ts:32]:
```suggestion
     public selectCode(isHSN: any): void {
```

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html [hunk 10]
```diff
@@ -1,45 +1,47 @@
...
-                        <label for="combo-option" class="radio-custom-label">Combo</label>
-                    </span>
-                    <span class="p-0 mr-3">
-                        <input id="bulk-creation-option" class="radio-custom cp ml-0" type="radio"
-                            name="selectCreateOption" [value]="">
-                        <label for="bulk-creation-option" class="radio-custom-label">Bulk Creation</label>
-                    </span>
+                <div class="radio-btn-wrapper mr-t15">
+                    <form [formGroup]="formGroupRadio" class="form-body">
+                        <span class="p-0 mr-3" >
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html:11]:
```suggestion
                        <span class="p-0 mr-3">
```

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html [hunk 11]
```diff
@@ -462,41 +464,48 @@
...
+                                    <div class="row">
                                         <div class="col-sm-6 col-lg-6">
-                                            <input id="hsn-code" class="form-control" type="text">
+                                            <span class="p-0 mr-3" (click)="selectCode(true)">
+                                                <input id="hsn-code-option" class="radio-custom cp ml-0" type="radio" name="selectHsnCode"
+                                                    [value]="">
+                                                <label for="hsn-code-option" class="radio-custom-label">HSN Code</label>
+                                            </span>
+                                            <span class="p-0" (click)="selectCode(false)">
+                                                <input id="hsc-code-option" class="radio-custom cp ml-0" type="radio" name="selectHsnCode" [value]="">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html:485]:
```suggestion
                                                <input id="sac-code-option" class="radio-custom cp ml-0" type="radio" name="selectHsnCode" [value]="">
```

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html [hunk 12]
```diff
@@ -462,41 +464,48 @@
...
+                                                <label for="hsc-code-option" class="radio-custom-label">SAC Code</label>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html:486]:
```suggestion
                                                <label for="sac-code-option" class="radio-custom-label">SAC Code</label>
```

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html [hunk 13]
```diff
@@ -976,14 +984,30 @@
...
+                                    <div class="col-sm-12 col-md-6 col-lg-6">
+                                        <sh-select id="select-under-group" placeholder="Select Group"></sh-select>
+                                    </div>
+                                </div>
+                                <div class="row mr-b15">
+                                    <div class="col-sm-12 col-md-4 col-lg-6">
+                                        <label>Unique Name<span class="red-text">*</span></label>
+                                    </div>
+                                    <div class="col-sm-12 col-md-6 col-lg-6">
+                                    <input id="combo-Unique-name" class="form-control" type="text" placeholder="EmtUniqueer Name">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html:1008]:
```suggestion
                                    <input id="combo-unique-name" class="form-control" type="text" placeholder="EmtUniqueer Name">
```

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.ts [hunk 14]
```diff
@@ -12,9 +14,55 @@ export class CreateNewInventoryComponent implements OnInit {
...
+    public isHSN: boolean = true;
+    /* this will store product boolean value */
+    public isProduct: boolean = true;
+    public isService: boolean = false;
+    public isCombo: boolean = false;
+    public isBulkCreation: boolean = false
+    public formGroupRadio: FormGroup;
+    private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
+
+    constructor(private _fb: FormBuilder
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.ts:27]:
```suggestion
    constructor(private fb: FormBuilder
```

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.ts [hunk 15]
```diff
@@ -12,9 +14,55 @@ export class CreateNewInventoryComponent implements OnInit {
...
+                this.isProduct = false;
+                this.isService = false;
+                this.isCombo = false;
+                this.isBulkCreation = true;
+            }
+        });
         /* added image path */
         this.imgPath = (isElectron || isCordova) ? 'assets/images/' : AppUrl + APP_FOLDER + 'assets/images/';
     }
+    selectCode(isHSN) {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.ts:64]:
```suggestion
    public selectCode(isHSN: any): void {
```

File: apps/web-giddh/src/app/new-inventory/component/create-new-item/create-new-item.component.ts [hunk 16]
```diff
@@ -24,4 +26,8 @@ export class CreateNewItemComponent implements OnInit {
         /* added image path */
         this.imgPath = (isElectron || isCordova) ? 'assets/images/' : AppUrl + APP_FOLDER + 'assets/images/';
     }
+    /* It will show/hide hsn code field */
+    public selectCode(isHSN) {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/create-new-item/create-new-item.component.ts:30]:
```suggestion
    public selectCode(isHSN: any): void {
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/ledger/ledger.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html:268]: Updated

File: apps/web-giddh/src/app/new-inventory/component/adjust-group/adjust-group.component.html (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/adjust-group/adjust-group.component.html:3]: Updated

File: apps/web-giddh/src/app/new-inventory/component/create-combo/create-combo.component.html (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-combo/create-combo.component.html:5]: Updated

File: apps/web-giddh/src/app/new-inventory/component/create-new-item/create-new-item.component.html (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-new-item/create-new-item.component.html:5]: Updated

File: apps/web-giddh/src/app/new-inventory/component/create-unit/create-unit.component.html (hunk 5 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-unit/create-unit.component.html:5]: Updated

File: apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.scss (hunk 6 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.scss:22]: Updated

File: apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts (hunk 7 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.ts:105]: Done

File: apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.scss (hunk 8 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.scss:26]: Done

File: apps/web-giddh/src/app/new-inventory/component/create-new-group/create-new-group.component.ts (hunk 9 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-new-group/create-new-group.component.ts:32]: Done

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html (hunk 10 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html:11]: Done

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html (hunk 11 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html:485]: Done

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html (hunk 12 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html:486]: Done

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html (hunk 13 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html:1008]: Done

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.ts (hunk 14 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.ts:27]: Done

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.ts (hunk 15 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.ts:64]: Done

File: apps/web-giddh/src/app/new-inventory/component/create-new-item/create-new-item.component.ts (hunk 16 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-new-item/create-new-item.component.ts:30]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Review by aditi-tiwari17 — APPROVED** [review]

**Pull request merged**